src/journal/views.py

```diff
@@ -1706,6 +1706,9 @@ def manage_archive_article(request, article_id):
         note_form = submission_forms.PublisherNoteForm(instance=publisher_note)
         note_forms.append(note_form)
 
+    assigned_editors = [assignment.editor for assignment in
```

Comment:
If splitting across lines it might be better as:

```
assigned_editors = [
    assignment.editor for assignment in review_models.EditorAssignment.objects.filter(
        article=article,
    )
]
```